CONTRIBUTING.md

```diff
@@ -19,8 +19,37 @@ limitations under the License.
 # Contributing
 
 Welcome to Elyra! If you are interested in contributing to the [Elyra code repo](README.md)
-then checkout the [Contributor's Guide](https://github.com/elyra-ai/community/blob/master/contributing.md).
+then checkout the [Contributor's Guide](https://github.com/elyra-ai/community/blob/master/contributing.md). 
 
 The [Elyra community repo](https://github.com/elyra-ai/community) contains information on how the community
 is organized and other information that is pertinent to contributing.
 
+### Getting Started
+
+Before contributing, we highly suggest reading the [Code of Conduct](https://github.com/elyra-ai/community/blob/master/code-of-conduct.md) & 
+[Contributor Repo Setup Documentation](https://elyra.readthedocs.io/en/latest/developer_guide/documentation.html). 
+Once completed, it's encouraged that you look under the [Issues](https://github.com/elyra-ai/elyra/issues) tab for contribution opportunites.
+For those new to the project or OpenSource workflows, we suggest looking under:
+
+- Issues tagged as [Good First Use](https://github.com/elyra-ai/elyra/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22good+first+issue%22)
+- Issues tagged as [Help Wanted](https://github.com/elyra-ai/elyra/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22help+wanted%22 )
+
+If you are planning to contribute to source code, the following can help you set up your [development environment](https://elyra.readthedocs.io/en/latest/developer_guide/development-workflow.html).
+
+
+#### Suggested Small to Medium Sized Projects
```

Comment:
This is a fluent list, so I would suggest creating a tag on github and linking to that, similar to how we did above?

Reply:
Hi Luciano, I'm not sure how to create a tag for these issues. Could you show me/create the necessary tags for me so that I could link them?

Reply:
I did create the label, and started adding to some issues:
https://github.com/elyra-ai/elyra/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+project%22

Reply:
Resolved. Let me know if you have any suggestions!